### PR TITLE
[CLI] a warm welcome message included :)

### PR DESCRIPTION
**Pull request by billy1624**
## PR Info

- Closes https://github.com/SeaQL/sea-orm/issues/1253

## Changes

- [x] Added more info to the welcome message of `sea-orm-cli`

```
sea-orm/sea-orm-cli on  master [$!?] is 📦 v0.10.3 via 𝗥 v1.66.1 via 🅒 base
➜ cargo r
   Compiling sea-orm-cli v0.10.3 (/Applications/MAMP/htdocs/sea-orm/sea-orm-cli)
    Finished dev [unoptimized + debuginfo] target(s) in 8.90s
     Running `target/debug/sea-orm-cli`
sea-orm-cli 0.10.3
Billy Chan <ccw.billy.123@gmail.com>

   ____                 ___   ____   __  __        /\
  / ___|   ___   __ _  / _ \ |  _ \ |  \/  |      {.-}
  \___ \  / _ \ / _` || | | || |_) || |\/| |     ;_.-'\
   ___) ||  __/| (_| || |_| ||  _ < | |  | |    {    _.}_
  |____/  \___| \__,_| \___/ |_| \_\|_|  |_|     \.-' /  `,
                                                  \  |    /
  An async & dynamic ORM for Rust                  \ |  ,/
  ===============================                   \|_/

  Getting Started!
    - documentation: https://www.sea-ql.org/SeaORM
    - step-by-step tutorials: https://www.sea-ql.org/sea-orm-tutorial
    - integration examples: https://github.com/SeaQL/sea-orm/tree/master/examples
    - cookbook: https://www.sea-ql.org/sea-orm-cookbook

  Join our Discord server to chat with others in the SeaQL community!
    - invitation link: https://discord.com/invite/uCPdDXzbdv

  If you like what we do, consider starring, commenting, sharing and contributing!

USAGE:
    sea-orm-cli [OPTIONS] <SUBCOMMAND>

OPTIONS:
    -h, --help       Print help information
    -v, --verbose    Show debug messages
    -V, --version    Print version information

SUBCOMMANDS:
    generate    Codegen related commands
    help        Print this message or the help of the given subcommand(s)
    migrate     Migration related commands
```

### CODE REVIEW

File: sea-orm-cli/src/cli.rs [hunk 1]
```diff
@@ -1,7 +1,31 @@
 use clap::{ArgEnum, ArgGroup, Parser, Subcommand};
 
 #[derive(Parser, Debug)]
-#[clap(version)]
+#[clap(
+    version,
```
Comment (tyt2y3) [inline sea-orm-cli/src/cli.rs:5]: Can we print the version below the about but before the actual help?

Comment (billy1624) [reply sea-orm-cli/src/cli.rs:5]: Yep. That's possible

**Comment by billy1624** [comment]
Now it looks like this. I put your name as well :)

```
sea-orm/sea-orm-cli on  cli-welcome-msg [$!] is 📦 v0.10.3 via 𝗥 v1.66.1 via 🅒 base took 11s
➜ cargo r -- --help
    Finished dev [unoptimized + debuginfo] target(s) in 0.72s
     Running `target/debug/sea-orm-cli --help`
sea-orm-cli 0.10.3

   ____                 ___   ____   __  __        /\
  / ___|   ___   __ _  / _ \ |  _ \ |  \/  |      {.-}
  \___ \  / _ \ / _` || | | || |_) || |\/| |     ;_.-'\
   ___) ||  __/| (_| || |_| ||  _ < | |  | |    {    _.}_
  |____/  \___| \__,_| \___/ |_| \_\|_|  |_|     \.-' /  `,
                                                  \  |    /
  An async & dynamic ORM for Rust                  \ |  ,/
  ===============================                   \|_/

  Getting Started!
    - documentation: https://www.sea-ql.org/SeaORM
    - step-by-step tutorials: https://www.sea-ql.org/sea-orm-tutorial
    - integration examples: https://github.com/SeaQL/sea-orm/tree/master/examples
    - cookbook: https://www.sea-ql.org/sea-orm-cookbook

  Join our Discord server to chat with others in the SeaQL community!
    - invitation link: https://discord.com/invite/uCPdDXzbdv

  If you like what we do, consider starring, commenting, sharing and contributing!


USAGE: sea-orm-cli [OPTIONS] <SUBCOMMAND>

OPTIONS:
    -h, --help       Print help information
    -v, --verbose    Show debug messages
    -V, --version    Print version information

SUBCOMMANDS:
    generate    Codegen related commands
    migrate     Migration related commands
    help        Print this message or the help of the given subcommand(s)

AUTHORS:
    Chris Tsang <chris.2y3@outlook.com>, Billy Chan <ccw.billy.123@gmail.com>
```

**Review by tyt2y3 — APPROVED** [review]
Kind of cool. Didn't know clap is so flexible

**Pull request merged**